dotcom-rendering/src/components/CrosswordInstructions.tsx

```diff
@@ -0,0 +1,35 @@
+import { css } from '@emotion/react';
+import {
+	textEgyptian17,
+	textEgyptianBold17,
+} from '@guardian/source/foundations';
+import sanitise from 'sanitize-html';
+
+const instructionsStyles = css`
+	${textEgyptian17};
+	white-space: pre-line;
+
+	i {
+		font-style: italic;
+	}
+
+	b {
+		font-weight: bold;
+	}
+`;
+
+const headerStyles = css`
+	${textEgyptianBold17};
+	white-space: pre-line;
+`;
+
+export const CrosswordInstructions = ({
+	instructions,
+}: {
+	instructions: string;
+}) => (
+	<div css={instructionsStyles}>
+		<strong css={headerStyles}>Special instructions: </strong>
+		<span dangerouslySetInnerHTML={{ __html: sanitise(instructions) }} />
```

Comment:
Crossword instructions can contain markup to apply formatting. eg.
https://www.theguardian.com/crosswords/quick-cryptic/42

As it's from an internal source perhaps we don't need to sanitise it?